raven/models/emulators.py

```diff
@@ -59,7 +59,7 @@ class GR4JCN_OST(Ostrich, GR4JCN):
     def __init__(self, *args, **kwds):
         super().__init__(*args, **kwds)
         self.rvi.suppress_output = True
-        self.txt = Ost(algorithm='DSS',
+        self.txt = Ost(algorithm='DDS',
```

Comment:
Another way to set the default ostrich algorithm would be to set `self.algorithm="DDS"` in the class `rv.OST`.

Reply:
Do you suggest we do that? or do we leave it as is?